src/mmw/js/src/core/views.js

```diff
@@ -673,6 +679,19 @@ var MapView = Marionette.ItemView.extend({
             }
         });
     },
+
+    renderFocusBounds: function() {
```

Comment:
Consider taking `focusBounds` out of the map model, and moving `renderFocusBounds` into the data source view. This would better match how we display the water quality catchment shapes.